app/controllers/api/base_controller.rb

```diff
@@ -21,5 +21,13 @@ def authenticate_user
       @current_api_user = try_spree_current_user
       super
     end
+
+    # Allows API access without authentication, but only for OFN controllers which inherit
+    # from BaseController. Actions that require authentication should all use #authorize!
```

Comment:
maybe change to "from Api::BaseController (that inherits from Spree::Api::BaseController)"